vite-project/src/strategypage/RadarChart.tsx

```diff
@@ -0,0 +1,266 @@
+import React, { useRef, useEffect } from "react";
+
+export interface point {
```

Comment:
Why is this a stupid interface instead of a class with `.add(Point)`, `.scale(number)`, `.rotate(angle)`, ...

Reply:
what?